faq/pathogen-subs-guide.rst

```diff
@@ -209,23 +210,33 @@ the library description. This can be, for example, the name and/or URL to a spec
 Submit Assembled Sequences
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+The instructions below provide a quick guide to submitting a completed isolate pathogen genome assembly.
+This type of submission is classed as 'clone or isolate' **ASSEMBLY_TYPE** for the ENA submissions services. For submission of
+other types of nucleotide assembly data, please see the submission options `here <https://ena-docs.readthedocs.io/en/latest/submit/assembly.html#submission-options>`_.
+For submission of targeted sequences, please refer to the `targeted sequence submissions guide <https://ena-docs.readthedocs.io/en/latest/submit/sequence.html#how-to-submit-targeted-sequences>`_.
 
-The instructions here provide tips for submitting microbial pathogen assemblies. For genome assembly submission,
-Webin-CLI (command line interface) needs to be used. The guide for downloading and using Webin-CLI is `here <https://ena-docs.readthedocs.io/en/latest/submit/general-guide/webin-cli.html#webin-cli-submission>`_.
+For genome assembly submission, Webin-CLI (command line interface) needs to be used. The guide for downloading
+and using Webin-CLI is `here <https://ena-docs.readthedocs.io/en/latest/submit/general-guide/webin-cli.html#webin-cli-submission>`_.
 
 .. tip::
-   For submission of targeted sequences, please refer to the `targeted sequence submissions guide <https://ena-docs.readthedocs.io/en/latest/submit/sequence.html#how-to-submit-targeted-sequences>`_.
+   **A note on assembly levels**
```

Comment:
I think this:
```
.. tip:: A note on assembly levels
    This guide ........ etc

``` 
would result in the tip box having a custom heading (https://sphinxawesome.xyz/demo/notes/). It's absolutely fine as it is, but might be nice to see how it looks?